lib/reply.js

```diff
@@ -307,6 +301,12 @@ function handleError (reply, error, cb) {
     return
   }
 
+  if (statusCode >= 500) {
+    res.log.error({ req: reply.request.raw, res: res, err: error }, error && error.message)
+  } else if (statusCode >= 400) {
+    res.log.info({ req: reply.request.raw, res: res, err: error }, error && error.message)
```

Comment:
I'm not convinced here: log the request every time could be onerous. Or not?

Reply:
This was just pre-existing behavior moved around, if you would like to discuss that open a new issue/p.

Reply:
I'm not sure logging the response object is correct here. I can understand the request being logged, but I really think only the error needs to be logged.